index.html

```diff
@@ -6,28 +6,25 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>personalportolio</title>
     <link rel="stylesheet" href="style.css" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;400&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <header>
+      <div class="logo"><h1 id="mylogo">My Logo</h1></div>
```

Comment:
- [ ] Kindly make this logo a link, normally logo on a website links to the index page